DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Extensions/HttpResponseExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubExtention.Preferences.WebApi.Extensions
+{
+    public static class HttpResponseExtensions
+    {
+        public static string GetContentType(this Task<HttpResponseMessage> response, string contentKey)
```

Comment:
Can you explain why this method is not extension method with a single responsibility?